File: LibLink_API/LibLink_API/Models/Review.cs
namespace LibLink_API.Models
{
    public class Review
    {
        public int ReviewID { get; set; }

        public int UserID { get; set; }

        public int BookID { get; set; }

        public int Rating { get; set; }

        public string Comment { get; set; } = string.Empty;

        public DateTime CreatedDate { get; set; } = DateTime.UtcNow;

        public User User { get; set; } = null!;

        public Book Book { get; set; } = null!;
    }
}

File: app/src/main/java/com/example/liblinkapp/models/Review.kt
package com.example.liblinkapp.models

import com.google.gson.annotations.SerializedName

data class Review(
    @SerializedName("reviewID")
    val reviewID: Int = 0,

    @SerializedName("userID")
    val userID: Int,

    @SerializedName("bookID")
    val bookID: Int,

    @SerializedName("rating")
    val rating: Int,

    @SerializedName("comment")
    val comment: String,

    @SerializedName("createdDate")
    val createdDate: String? = null,

    @SerializedName("user")
    val user: User? = null
)